backend/src/middleware/helpers/email/sendMail.ts

```diff
@@ -7,32 +7,12 @@
 import { createTransport } from 'nodemailer'
 import { htmlToText } from 'nodemailer-html-to-text'
 
-import CONFIG from '@config/index'
+import CONFIG, { nodemailerTransportOptions } from '@config/index'
 import { cleanHtml } from '@middleware/helpers/cleanHtml'
 
-const hasEmailConfig = CONFIG.SMTP_HOST && CONFIG.SMTP_PORT
-const hasAuthData = CONFIG.SMTP_USERNAME && CONFIG.SMTP_PASSWORD
-const hasDKIMData =
-  CONFIG.SMTP_DKIM_DOMAINNAME && CONFIG.SMTP_DKIM_KEYSELECTOR && CONFIG.SMTP_DKIM_PRIVATKEY
+const hasEmailConfig = nodemailerTransportOptions.host && nodemailerTransportOptions.port
```

Comment:
I decided to export `nodemailerTransportOptions` directly in `src/config/index.ts` because all the settings are used together except here and because of complexity.

Reply:
nvm i missed the file above